conda_smithy/templates/jenkinsfile.tmpl

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/groovy
+pipeline {
+  agent none
+  stages {
+{%- block env_test %}
+   stage('build') {
+      parallel {
+{%- for config, platform, config_docker_image  in configs %}
+        stage('{{ config }}') {
+          agent {
+            label '{{ config_docker_image }}'
+          }
+          environment {
+            CONFIG="${STAGE_NAME}"
+            FEEDSTOCK_ROOT="${WORKSPACE}"
+            RECIPE_ROOT="${FEEDSTOCK_ROOT}/recipe"
+            ARTIFACTS="${FEEDSTOCK_ROOT}/build_artifacts"
+          }
+          steps {
+            sh """#!/bin/bash -il
+               conda activate base
+               .circleci/build_steps.sh
```

Comment:
All the build-steps come from `.circleci/build_steps.sh` to address @isuruf's comment.